CacheRefresh.php

```diff
@@ -67,9 +48,8 @@ public function getMappings()
      * @param mixed $object            Object that was changed
      * @param array $changedProperties List of property paths (ex. ['status.description', 'userCount', ...])
      *
-     * @return array
      */
-    public function getUrlsToRefresh($object, $changedProperties)
+    public function getUrlsToRefresh($object, array $changedProperties): array
```

Comment:
Might be a good time to also rename this method. The urls are purged not refreshed, so at the very least this should be getUrlsToPurge.

Reply:
Method is now renamed to `getUrlsToPurge`.